### PR TITLE
Fixed URL and swagger Import

### PR DESCRIPTION
**Pull request by HiranyaKavishani**

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx [hunk 1]
```diff
@@ -198,6 +164,14 @@ class APIDefinition extends React.Component {
         this.setState({ swagger: formattedString, format: convertTo, convertTo: format });
     }
 
+    setSchemaDefinition=(swagger, graphQL) => {
```
Comment (dushansilva) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx]: There should be a Eslint issue here

Comment (HiranyaKavishani) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx]: It doesn't see any Eslint issue in here

Comment (harsha89) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx]: Are we good? Shall we merge @dushansilva

Comment (dushansilva) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx]: I checked locally for some reason that eslint rule is being ignore for this file only, not sure why, typically there should be a space before and after the = sign

Comment (HiranyaKavishani) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx]: prod build was pass

Comment (HiranyaKavishani) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx]: Prod build was passed

**Pull request merged**